examples/active_replication/controller_operator.py

```diff
@@ -0,0 +1,47 @@
+import time
+
+from erdos.data_stream import DataStream
+from erdos.message import Message
+from erdos.op import Op
+from erdos.timestamp import Timestamp
+from erdos.utils import setup_logging
+
+import flux_utils
+
+
+class ControllerOperator(Op):
+    def __init__(self,
+                 name,
+                 fail_timeout=1,
+                 log_file_name=None):
+        super(ControllerOperator, self).__init__(name)
+        self._logger = setup_logging(self.name, log_file_name)
+        self._node_failed = False
+        self._timeout = fail_timeout
+
+    @staticmethod
+    def setup_streams(input_streams):
+        return [DataStream(name='controller_stream',
+                           labels={'control_stream': 'true'})]
+
+    def execute(self):
+        # TODO(ionel): We should send the failure message on a failed stream,
+        # and get a reply back so that we know the node is not processing or
+        # issuing any new messages. Otherwise, we run into the risk of having
+        # inconsistent state because we can't guarantees in-order delivery
+        # across operators.
+        
+        # Send failure message.
+        time.sleep(3)
```

Comment:
Can you please parametrize the time after which to insert a failure using an operator parameter?